ui/src/components/HookForm/index.jsx

```diff
@@ -182,7 +183,7 @@ export default class HookForm extends Component {
   state = {
     hook: null,
     hookLastFires: null,
-    pattern: '#',
+    routingKeyPattern: '#',
```

Comment:
The server expects this property name to be `pattern`. I don't think this will work atm.

Reply:
It turns out the hook service [expects](https://schemas.taskcluster.net/hooks/v1/bindings.json#) it to be `routingKeyPattern`. Sorry for that. Now that you have the server setup, are you able to add a hook to http://localhost:5080/hooks/garbage/abc456 and see if it saves successfully?

Reply:
When trying to save i was getting 401 Authentication failed. 
I thought it needs some level of access to save it. So couldn't validate it

Reply:
How were you signing in? I can provide you with the required level of access.

Reply:
You will need to use the manual siginin `./taskcluster signin --name taskcluster-web-server`. The taskcluster binary can be downloaded from https://github.com/taskcluster/taskcluster-cli#installation. The `taskcluster signin` command will open a browser where you will be able to login then upon logging in, the binary will output the credentials in the terminal. Those credentials is what you need to login.

Reply:
I am doing this, in browser I used auth0 login and used those credentials to login in UI

Reply:
What is the exact error you are seeing when you try to create/update a hook?

Reply:
Do you get the same error when creating/updating a hook via https://tools.taskcluster.net/?